src/main/java/edu/rosehulman/quota/javasdk/Quota.java

```diff
@@ -0,0 +1,24 @@
+package edu.rosehulman.quota.javasdk;
+
+public class Quota {
+
+  private Partner partner;
+  private Product product;
+  private User user;
+  private String quotaId;
+
+  public Quota(Partner partner, Product product, User user, String quotaId) {
+    this.partner = partner;
+    this.product = product;
+    this.user = user;
+    this.quotaId = quotaId;
+  }
+
+  public boolean increment() throws QuotaReachedException {
```

Comment:
Should probably accept an argument to say how much to increment by or add an overloaded increment(int amount)

Reply:
That is a completely different feature I think. I thought this feature was to increment by the default amount.